1-exercises/D-array-filter/exercise.js

```diff
@@ -15,8 +17,7 @@ let mentors = ["Daniel", "Irina", "Mozafar", "Luke"];
 
 let pairs = pairsByIndex.map(function (indexes) {
   let student = students[indexes[0]];
```

Comment:
if you want to add mentors here, how can you do that and what will your function return?